### ISSUE TITLE
Add unit tests for memmove()

### ISSUE THREAD

**Issue by anton-bondarev** [embox/embox#1858]
Add some test cases for standard memmove()

**Comment by kouamdo** [comment]
Hey , Hi, I would like to be assigned to this task.

thank you ...

**Comment by Deryugin** [comment]
@kouamdo ok!

**Comment by kouamdo** [comment]
thank you for all

**Comment by kouamdo** [comment]
I'm starting to solve that task

**Comment by kouamdo** [comment]
wow , look :

```
void memmove( void *src , void *dest , size_t size )
{
  strcpy(dest , src , size) ;
  free(src) ;
}
```
i think it can be fine .

What do you think ??

**Comment by anton-bondarev** [comment]
@kouamdo, First, you should create PR :)

**Comment by anton-bondarev** [comment]
Closed with #2072
